main/cryptsetup-scripts/update.py

```diff
@@ -1,5 +1,5 @@
-pattern = r"cryptsetup\ Debian\ release\ 2:([\d.\-~rc]+)"
-ignore = ["*rc*", "*~*"]
+pattern = r"cryptsetup\ Debian\ release\ 2:([\d.\-~rc\+exp]+)"
```

Comment:
How about we match all letters maybe, or even just match the version in URLs like https://salsa.debian.org/cryptsetup-team/cryptsetup/-/tags/debian%2F2%252.7.1-1+exp?

Reply:
hm not sure what the benefit would be but maybe i'm too tired. does the existing check fail in some case?

Reply:
If you mean the original, it matches experimental versions (like the version in the link above).

Reply:
hm seems ok then, sure